modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -31,27 +31,38 @@ The `SMA2` connector is bidirectional.
 ====
 
 Set `spec.profile.plugins.e810.ublxCmds` parameters to configure the GNSS clock in the `PtpConfig` custom resource (CR).
+
+[IMPORTANT]
+====
+You must configure separate delay offset values to counteract 1PPS signal delay and T-GM antenna cable signal delay.
+To calculate optimal offset values that do not introduce constant time errors, make precise measurements of the 1PPS cable and GNSS antenna cable signal delays.
+Red Hat cannot assist in this measurement or provide any values for the required delay offsets.
+====
+
 Each of these `ublxCmds` stanzas correspond to a configuration that is applied to the host NIC by using `ubxtool` commands.
 For example:
 
 [source,yaml]
 ----
 ublxCmds:
-  - args: #ubxtool -P 29.20 -z CFG-HW-ANT_CFG_VOLTCTRL,1
+  - args:
       - "-P"
       - "29.20"
       - "-z"
       - "CFG-HW-ANT_CFG_VOLTCTRL,1"
+      - "-z"
+      - "CFG-TP-ANT_CABLEDELAY,<calculated_delay_offset>"<1>
     reportOutput: false
 ----
+<1> Calculated T-GM antenna delay offset in microseconds, for example `25`.
```

Comment:
```suggestion
<1> Pre-calibrated T-GM antenna delay offset in nanoseconds, for example `25`.
```

Reply:
Pre-calibrated or pre-measured, or something of sorts. The process they do for the delays must contain measurement anr / or calibration